scripts/slack/notify_deploy.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/python3
+
+import sys
+import requests
+import json
+
+image_name = sys.argv[2]
```

Comment:
what's in argv[1]?

Reply:
that should definitely be 1 and 2 index below, fixing now.

Reply:
If we look at the usage of this notify_deploy script:
```sh
./scripts/slack/notify_deploy.py $LATEST_TAG dev
```
I think argv[1] would be the name of the tag, and argv[2] the deployment env... so argv[3] doesn't exist, right? I think the indices are 1 too many

Reply:
that is fixed now!